ci/build_test_OnCommit.groovy

```diff
@@ -115,6 +124,9 @@ pipeline {
         }
         stage("Release image and tests in parallel") {
           when { expression { image_build_needed == "true" } }
+          options {
```

Comment:
2 HOURS here?

Reply:
Yes - I just put initially 4 minutes to verify if it works.